crates/solidity/inputs/schema/grammar/04-statements/01-blocks/productions.yml

```
@@ -25,6 +25,23 @@
   kind: "Parser"
   versioned:
     0.4.11:
+      choice:
+        - reference: "Block"
+        - reference: "SimpleStatement"
+        - reference: "IfStatement"
+        - reference: "ForStatement"
+        - reference: "WhileStatement"
+        - reference: "DoWhileStatement"
+        - reference: "ContinueStatement"
+        - reference: "BreakStatement"
+        - reference: "ReturnStatement"
+        - reference: "EmitStatement"
+        - reference: "ThrowStatement"
+        - reference: "RevertStatement"
+        - reference: "DeleteStatement"
+        - reference: "AssemblyStatement"
+    0.5.0:
+      # removed: "throw" statement
```

nit suggestion to make it easier to copy/search:

```suggestion
      # removed: "ThrowStatement"
```

---

same for others.

---

Same for others.

---

done